src/super_gradients/common/sg_loggers/base_sg_logger.py

```diff
@@ -247,7 +247,7 @@ def add_checkpoint(self, tag: str, state_dict: dict, global_step: int = None):
 
         path = os.path.join(self._local_dir, name)
         torch.save(state_dict, path)
-        if tag == 'ckpt_best.pth':
+        if 'best' in tag:
```

Comment:
In the callback we now change the new attribute "best_ckpt_name", so that in the QAT part we won't override our best checkpoints from earlier. We still want to see the message that we got the best results so far though.